ui/desktop/electron-app/package.json

```diff
@@ -21,15 +21,16 @@
         "ignore": [
           "/ember-test(/|$)",
           "/tests(/|$)"
-        ]
+        ],
+        "name": "Boundary",
+        "appBundleId": "com.electron.boundary",
+        "appVersion": "0.0.1",
+        "appCopyright": "Copyright © 2021 HashiCorp, Inc.",
+        "osxSign": {
+          "identity": "Developer ID Application: * (*)"
+        }
```

Comment:
@mdeggies,  @shore: I converted electron-builder config changes in https://github.com/hashicorp/boundary-ui/pull/424 into electron-forge config. @randallmorey and I figured out a way to get production desktop client to packaged into a DMG successfully using electron-forge. As electron-forge handles static assets for us easily, we'd like to use it going forward instead of electron-builder.

We can now generate a production desktop client DMG using `yarn build:ui:desktop` command from root of this repo. Let us know if either of you need any additional details. Ty for your patience while we figure this out at our end.

Reply:
cc @99 who is working on this! :-)